### PR TITLE
Migrate service_extension_widgets.dart to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/ui/service_extension_widgets.dart [hunk 1]
```diff
@@ -893,8 +894,8 @@ class ServiceExtensionRichTooltip extends StatelessWidget {
               Align(
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/ui/service_extension_widgets.dart:894]: only show this align widget if both description.tooltipUrl != null and description.gaScreenName != null

File: packages/devtools_app/lib/src/ui/service_extension_widgets.dart [hunk 2]
```diff
@@ -76,7 +76,7 @@ class _ServiceExtensionButtonGroupState
       // VMServiceManager.
       final extensionName = extension.description.extension;
       // Update the button state to match the latest state on the VM.
-      final state = serviceManager.serviceExtensionManager
+      final state = serviceManager.serviceExtensionManager!
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/lib/src/ui/service_extension_widgets.dart:79]: done

File: packages/devtools_app/lib/src/ui/service_extension_widgets.dart [hunk 3]
```diff
@@ -485,7 +485,7 @@ class _ServiceExtensionCheckboxState extends State<ServiceExtensionCheckbox>
   Widget build(BuildContext context) {
     return ValueListenableBuilder(
       valueListenable: extensionAvailable,
-      builder: (context, available, _) {
+      builder: (context, bool available, _) {
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/lib/src/ui/service_extension_widgets.dart:488]: done

File: packages/devtools_app/lib/src/ui/service_extension_widgets.dart [hunk 4]
```diff
@@ -610,12 +612,12 @@ class _ServiceExtensionCheckboxGroupButtonState
...
       ),
     );
-    if (widget.tooltip != null && widget.tooltip.isNotEmpty) {
+    if (widget.tooltip != null && widget.tooltip!.isNotEmpty) {
       label = DevToolsTooltip(message: widget.tooltip, child: label);
     }
     return ValueListenableBuilder(
       valueListenable: _enabled,
-      builder: (context, enabled, _) {
+      builder: (context, bool enabled, _) {
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/lib/src/ui/service_extension_widgets.dart:620]: done

File: packages/devtools_app/lib/src/ui/service_extension_widgets.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/ui/service_extension_widgets.dart:894]: done

**Review by kenzieschmoll — APPROVED** [review]

**Comment by polina-c** [comment]
tested

**Pull request merged**